GitUI/CommandsDialogs/BrowseDialog/DashboardControl/UserRepositoriesList.cs

```diff
@@ -857,7 +891,10 @@ private void OnDragEnter(object sender, DragEventArgs e)
             }
         }
 
-        // returns false only if no repository is selected
+        /// <summary>
+        /// Tries to open the currently selected repository
+        /// </summary>
+        /// <returns>False if no repo is selected, true otherwise</returns>
         private bool TryOpenSelectedRepository()
```

Comment:
* Modify the signature as follows:
    ```suggestion
            private bool TryOpenSelectedRepository(Repository? repository)
    ```
* Update the existing callsites with `TryOpenSelectedRepository(GetSelectedRepository())`
* Remove `OpenRepository` and use `TryOpenSelectedRepository` instead